### PR TITLE
ADX-460-dataset-cloning

### PR DESCRIPTION
**Pull request by Manoj-nathwani**

### CODE REVIEW

**Comment by Manoj-nathwani** [comment]
@jonathansberry can you confirm I don't need to add my new field to `naomi_data/1_naomi_data.json`

**Review by jonathansberry — COMMENTED** [review]
Does it make sense to put a lot of the replication into a preset?

**Comment by jonathansberry** [comment]
> @jonathansberry can you confirm I don't need to add my new field to `naomi_data/1_naomi_data.json`

I think your feature should be included in every single data package schema.

**Comment by Manoj-nathwani** [comment]
@jonathansberry 
> Does it make sense to put a lot of the replication into a preset?

`¯\_(ツ)_/¯`
Is there a better way to go about it?
I guess if we want to stick to using JSON files for config then replication is probably unavoidable and ok

File: package_schemas/general_data/2_restricted_schema.json [hunk 1]
```diff
@@ -51,6 +51,14 @@
             "private": false,
             "visibility_classes": ["hidden"]
         },
+        {
+            "field_name": "org_to_allow_transfer_to",
+            "label": "Transfer to",
+            "preset": "org_to_allow_transfer_to",
+            "private": false,
+            "visibility_classes": ["hidden"],
+            "display_group": "Inputs to UNAIDS Estimates 2020"
```
Comment (jonathansberry) [inline package_schemas/general_data/2_restricted_schema.json]: So this line looks wrong to me... It's package specific, and in the wrong package.

File: package_schemas/general_data/2_restricted_schema.json [hunk 2]
```diff
@@ -51,6 +51,14 @@
             "private": false,
             "visibility_classes": ["hidden"]
         },
+        {
+            "field_name": "org_to_allow_transfer_to",
+            "label": "Transfer to",
+            "preset": "org_to_allow_transfer_to",
+            "private": false,
+            "visibility_classes": ["hidden"],
```
Comment (jonathansberry) [inline package_schemas/general_data/2_restricted_schema.json]: I'm thinking this line could go into the preset "org_to_allow_transfer_to".  Where have you defined that preset?  in ckanext-unaids/ckanext/unaids/presets.json?

File: package_schemas/general_data/2_restricted_schema.json [hunk 3]
```diff
@@ -51,6 +51,14 @@
             "private": false,
             "visibility_classes": ["hidden"]
         },
+        {
+            "field_name": "org_to_allow_transfer_to",
+            "label": "Transfer to",
```
Comment (jonathansberry) [inline package_schemas/general_data/2_restricted_schema.json]: This could probably go in the preset as well. (See comment below)

**Comment by tomeksabala** [comment]
I disagree. If some of the duplicate lines could be wrapped up into a preset then it should be done. Future changes into the field `org_to_allow_transfer_to` will be much easier to complete and review.

File: package_schemas/general_data/2_restricted_schema.json [hunk 4]
```diff
@@ -51,6 +51,14 @@
...
+            "preset": "org_to_allow_transfer_to",
+            "private": false,
```
Comment (jonathansberry) [inline package_schemas/general_data/2_restricted_schema.json]: This is a field I take it that you have copied across from the organisation field?  It is specific to the organisation form snippet.  Are you sure you need it for your feature?

**Review by jonathansberry — COMMENTED** [review]
Tried to comment on specific lines to break down my thoughts here.

**Review by jonathansberry — APPROVED** [review]

**Pull request merged**